docs/tutorials/getting-started.md

```diff
@@ -105,7 +103,7 @@ $ juju ssh microceph/0 "sudo ceph -s"
 
 The above shows a running Ceph cluster, however it displays a warning. Ceph warns because it by default expects 3 disks (OSDs in Ceph parlance) for storage, and we have not yet configured any.
```

Comment:
[* suggested rewording for specificity *] The above shows... -> The output above shows...